.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
+python: 3.4
```

Comment:
It means Travis creates a python v3.4 virtualenv
On 31 Aug 2015 23:10, "Arash Rouhani" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/spotify/luigi/pull/1179#discussion_r38365906>:
>
> > @@ -1,4 +1,5 @@
> >  language: python
> > +python: 3.4
>
> What does this do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/luigi/pull/1179/files#r38365906>.
>